themes/nord-color-theme.json

```
@@ -1215,6 +1215,45 @@
         "foreground": "#81A1C1"
       }
     },
+    {
+      "name": "[Rust] Entity types",
+      "scope": [
+        "source.rust entity.name.type"
+      ],
+      "settings": {
+        "foreground": "#8FBCBB"
+      }
+    },
+    {
+      "name": "[Rust] Macro",
+      "scope": [
+        "meta.macro.rust"
+      ],
+      "settings": {
+        "fontStyle": "bold",
+        "foreground": "#88C0D0"
+      }
+    },
+    {
+      "name": "[Rust] Attributes",
+      "scope": [
+        "meta.attribute.rust",
+        "meta.attribute.rust punctuation",
+        "meta.attribute.rust keyword.operator"
+      ],
+      "settings": {
+        "foreground": "#5E81AC",
+      }
+    },
```

```suggestion
    },
    {
      "name": "[Rust] Traits",
      "scope": "source.rust entity.name.type.trait",
      "settings": {
        "fontStyle": "bold"
      }
    },
```

Lets also ensure to use the correct highlighting for [Traits][1] (I know these language elements as _Interface_, e.g. from my main language [Go][2]) by making the font bold.

[1]: https://doc.rust-lang.org/book/ch10-02-traits.html
[2]: https://go.dev